### DISCUSSION TITLE
online radio

### DISCUSSION THREAD

**Discussion by KurTruC** [junguler/m3u-radio-music-playlists#6] [General]
Hello,

thank you for your work, would it be possible to add 1fm absolute top 40 Classic to your list please

best regard

**Comment by junguler** [comment]
hello, i've found it, just copy these 3 lines to a text file and save it as .m3u
```
#EXTM3U
#EXTINF:-1 , 1.FM - Absolute TOP 40 Classic
http://strm112.1.fm/top40ballads_mobile_mp3
```
several other 1.FM streams are also included in this repo, specifically in the [radio.net](https://github.com/junguler/m3u-radio-music-playlists/tree/main/radio.net) folder, check the [---everything-full](https://github.com/junguler/m3u-radio-music-playlists/raw/main/radio.net/---everything-full.m3u) file if you are interested

hope it helps, have a great rest of your day/night

**Comment by KurTruC** [comment]
thanks for all, but the link doesn't work

have a good day :)

**Comment by junguler** [comment]
turns out that was a re-stream from a third party, sorry about that, here from the actual source
```
#EXTM3U
#EXTINF:-1 , 1.FM - Absolute TOP 40 Classic
https://strmreg.1.fm/top40ballads_mobile_mp3
```

**Reply by KurTruC** [reply]
thank you for everything, it's strange that the link doesn't work for me (Switzerland), even with a vpn.
thanks anyway for the link, wish you a good day

**Reply by KurTruC** [reply]
the link of this site works for me here https://tunein.com/radio/Absolute-Top-40-Classic-s98682/, but it's not an mp3 link ;(

**Comment by junguler** [comment]
i've checked your link and it sais it has regional restriction for me too, i will try to find an alternate link for you if i can, this is really annoying indeed, i'm hoping we are able to fix this issue soon

**Comment by junguler** [comment]
hello again, i've looked thru radio.net , onlineradiobox and streema and all of them used that same link i've posted

https://strmreg.1.fm/top40ballads_mobile_mp3

i don't know why it's not playing for you, have you tried opening it in a browser and see if it's playing or not? not really sure how to scrape 1.fm or tunein tho since they hide the actual stream link inside javascript code and i really don't know how to get to it

**Comment by KurTruC** [comment]
yes I tried on other browser and nothing is working for me

Thanks anyway for everything, maybe we'll find the link one day ;)

**Comment by junguler** [comment]
i see, give the solutions in this page a try, it might help, otherwise i'm out of ideas :(

https://support.google.com/chrome/thread/29505473?hl=en&msgid=29673696
